src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -213,6 +213,9 @@
   <data name="IDS_FeatureAsyncStreams" xml:space="preserve">
     <value>async streams</value>
   </data>
+  <data name="IDS_FeatureUnmanagedGenericStructs" xml:space="preserve">
```

Comment:
"Unmanaged constructed types".

Reply:
I chose this because I believed C# programmers don't know what a constructed type is, but a web search for "C# constructed types" easily yields an understandable answer, so I'll change the name and the value here as you suggest.

Reply:
Feeling a little less certain on this one upon revisiting-- will kick it back to Jared, at least the text of the message presented to users.

Reply:
The actual text will be something like "switch to C# 8", right?

Reply:
Changed to 'unmanaged constructed types'

---
In reply to: [245833032](https://github.com/dotnet/roslyn/pull/31602#discussion_r245833032) [](ancestors = 245833032)